interface-definitions/dns-forwarding.xml.in

```diff
@@ -105,6 +105,456 @@
+                                <description>IPv4 address</description>
+                              </valueHelp>
+                              <multi/>
+                              <constraint>
+                                <validator name="ipv4-address"/>
+                              </constraint>
+                            </properties>
+                          </leafNode>
+                          #include <include/dns/time-to-live.xml.i>
+                          #include <include/generic-disable-node.xml.i>
+                        </children>
+                      </tagNode>
+                      <tagNode name="aaaa">
+                        <properties>
+                          <help>"AAAA" record</help>
+                          <valueHelp>
+                            <format>text</format>
+                            <description>A DNS name relative to the root record</description>
+                          </valueHelp>
+                          <valueHelp>
+                            <format>@</format>
+                            <description>Root record</description>
+                          </valueHelp>
+                          <constraint>
+                            <regex>^([-_a-zA-Z0-9.]{1,63}|@)(?&lt;!\.)$</regex>
+                          </constraint>
+                        </properties>
+                        <children>
+                          <leafNode name="address">
+                            <properties>
+                              <help>IPv6 address [REQUIRED]</help>
+                              <valueHelp>
+                                <format>ipv6</format>
+                                <description>IPv6 address</description>
+                              </valueHelp>
+                              <multi/>
+                              <constraint>
+                                <validator name="ipv6-address"/>
+                              </constraint>
+                            </properties>
+                          </leafNode>
+                          #include <include/dns/time-to-live.xml.i>
+                          #include <include/generic-disable-node.xml.i>
+                        </children>
+                      </tagNode>
+                      <tagNode name="cname">
+                        <properties>
+                          <help>"CNAME" record</help>
+                          <valueHelp>
+                            <format>text</format>
+                            <description>A DNS name relative to the root record</description>
+                          </valueHelp>
+                          <valueHelp>
+                            <format>@</format>
+                            <description>Root record</description>
+                          </valueHelp>
+                          <constraint>
+                            <regex>^([-_a-zA-Z0-9.]{1,63}|@)(?&lt;!\.)$</regex>
```

Comment:
This does not allow wildcard records.

Should I create an issue/feature request on phabricator?

Reply:
Hey, this is a good callout. Feel free to open a task, there are definitely some future improvements I plan to make to this.